pkgs/development/python-modules/ypy-websocket/default.nix

```
@@ -18,8 +18,6 @@ buildPythonPackage rec {
   version = "0.12.4";
   format = "pyproject";
```

```suggestion
  pyproject = true;
```
as well as `build-system` and `dependecies` would be nice